python_modules/dagster-graphql/dagster_graphql/client/client.py

```diff
@@ -251,6 +262,8 @@ def submit_job_execution(
                 schema for this job. If it does not, the client will throw a DagsterGraphQLClientError with a message of
                 JobConfigValidationInvalid. Defaults to None.
             tags (Optional[Dict[str, Any]]): A set of tags to add to the job execution.
+            op_selection (Optional[Sequence[str]]): A list of ops to execute.
+            asset_selection (Optional[Sequence[CoercibleToAssetKey]]): A list of asset keys to execute.
```

Comment:
It's unfortunate that this does not align with https://docs.dagster.io/guides/build/assets/asset-selection-syntax/reference but that is its own beast to correc.t

We should be able to expand this in the future without breaking existing functionality